libraries/AP_Math/vector2.h

```diff
@@ -276,6 +276,19 @@ struct Vector2
     }
 };
 
+// check if all elements are zero
+template<> inline bool Vector2<float>::is_zero(void) const {
+    return (fabsf(x) < FLT_EPSILON) && (fabsf(y) < FLT_EPSILON);
+}
+
+template<> inline bool Vector2<double>::is_zero(void) const {
+#ifdef ALLOW_DOUBLE_MATH_FUNCTIONS
+    return (fabs(x) < DBL_EPSILON) && (fabs(y) < DBL_EPSILON);
```

Comment:
needs to be FLT_EPSILON (see above comment)

Reply:
Fixed